From: Bob Picco <bpicco@redhat.com>
Date: Wed, 29 Sep 2010 16:47:25 -0400
Subject: [usb] net/catc: change NIC's TX_MAX_BURST, fixes probe
Message-id: <20100929165157.4329.73611.sendpatchset@localhost.localdomain>
Patchwork-id: 28522
O-Subject: [RHEL5.6 PATCH] BZ637826  Change TX_MAX_BURST in USB NIC catc.c
Bugzilla: 637826
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

[RHEL5.6 PATCH] BZ637826  Change TX_MAX_BURST in USB NIC catc.c

This change reduces alloc_etherdev's size argument for reasons
stated in:
144e94883f6a158e7eca29e6e1bed9acf3f03ed2
. Without this change the USB NIC doesn't probe in RHEL5.6.

Signed-off-by: Bob Picco <bpicco@redhat.com>


diff --git a/drivers/usb/net/catc.c b/drivers/usb/net/catc.c
index be5f5e1..8412463 100644
--- a/drivers/usb/net/catc.c
+++ b/drivers/usb/net/catc.c
@@ -73,7 +73,7 @@ static const char driver_name[] = "catc";
 #define TX_TIMEOUT		(5*HZ)	/* Max time the queue can be stopped */
 #define PKT_SZ			1536	/* Max Ethernet packet size */
 #define RX_MAX_BURST		15	/* Max packets per rx buffer (> 0, < 16) */
-#define TX_MAX_BURST		15	/* Max full sized packets per tx buffer (> 0) */
+#define TX_MAX_BURST		13	/* Max full sized packets per tx buffer (> 0) */
 #define CTRL_QUEUE		16	/* Max control requests in flight (power of two) */
 #define RX_PKT_SZ		1600	/* Max size of receive packet for F5U011 */
 
